lib/mysqlService.js

```diff
@@ -16,6 +16,10 @@ class MysqlService {
      * @param {Logger} [logger] - instance of hapiest-logger
      */
     constructor(writeConnectionConfig, readConnectionConfig, logger) {
+
+        this._writeConnectionConfig = writeConnectionConfig;
```

Comment:
Do we need the entire configs to be saved on the instance?  If not, let's just save off the variable(s) we need, which from what I can tell looks like timezone.